after/syntax/yaml.vim

```diff
@@ -24,7 +24,7 @@ syn match yamlBlock "[\[\]\{\}\|\>]"
 syn region yamlComment	start="\#" end="$"
 syn match yamlIndicator	"#YAML:\S\+"
 
-syn region yamlString	start="\%(^\| \)\zs'" end="'\ze\%( \|$\)" skip="\\'"
+syn region yamlString	start="\(^\|\s\|\[\|\,\|\-\)'" end="'" skip="\\'"
```

Comment:
I think you still want the \zs before the `'` in the `start` section, to prevent the leading [ or , from being included in the highlight.

As per my picture, doesn't seem to work for me. I think this is because the [ character is clashing with the syntax for `yamlBlock` above.

Reply:
Indeed, but yamlBlock is useful.

I'll try to find a compromise ...